tests/manage/cluster/test_must_gather.py

```diff
@@ -0,0 +1,112 @@
+import os
+import logging
+
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import openshift_ops, ocp
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.utils import collect_ocs_logs
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-1583")
+class TestMustGather(ManageTest):
+
+    @pytest.fixture(autouse=True)
+    def init_ocp(self):
+        """
+        init OCP() object
+        """
+
+        self.ocs = openshift_ops.OCP()
+        self.ocp_obj = ocp.OCP()
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+
+        def finalizer():
+            must_gather_pods = pod.get_all_pods(selector=['app=must-gather'])
+            logger.info(f"must_gather_pods: {must_gather_pods} ")
+            if must_gather_pods:
+                for pod_to_del in must_gather_pods:
+                    self.ocp_obj.wait_for_delete(resource_name=pod_to_del)
+
+        request.addfinalizer(finalizer)
+
+    def test_must_gather(self):
+
+        # Make logs root directory
+        logger.info("Creating logs Directory")
+        directory = self.make_directory()
+        logger.info(f"Creating {directory} - Done!")
+
+        # Collect OCS logs
+        logger.info("Collecting Logs")
+        collect_ocs_logs(dir_name=directory, ocp=False)
+        logger.info("Collecting logs - Done!")
+
+        # Compare running pods list to "/pods" subdirectories
+        logger.info("Checking logs tree")
+        logs = self.get_log_directories(directory)
+        pods = self.get_ocs_pods()
+        logger.info(f"Logs: {logs}")
+        logger.info(f"pods list: {pods}")
+        assert set(sorted(logs)) == set(sorted(pods))
+
+        # Verify logs file are not empty
```

Comment:
where is that verification?

Reply:
currently only 1st test is ready. this one would be next